index.json

```diff
@@ -109,7 +109,9 @@
       "minitest"
     ],
     "requires": [
-      "minitest/test"
+      "minitest/test",
```

Comment:
Those requires where required to be able to load the constants used in the annotations file.